playbooks/roles/jenkins/tasks/05_tools.yml

```diff
@@ -16,6 +16,25 @@
   tags: [apt]
   apt_repository: repo='deb http://apt.postgresql.org/pub/repos/apt/ {{ ansible_distribution_release }}-pgdg main' state=present
 
+- name: Add key for Chrome repo
+  tags: [apt]
+  apt_key:
+    url: https://dl-ssl.google.com/linux/linux_signing_key.pub
+
+- name: Add Chrome repo to sources list
+  tags: [apt]
+  apt_repository:
+    repo: deb http://dl.google.com/linux/chrome/deb/ stable main
+    state: present
+
+- name: Install Chromedriver binary
+  tags: [apt]
+  unarchive:
+    src: "https://chromedriver.storage.googleapis.com/92.0.4515.43/chromedriver_linux64.zip"
```

Comment:
How will we keep these pinned versions up to date?

Reply:
I was going to add an item to the monthly [Jenkins spring clean](https://trello.com/c/yxgnpt4I/793-do-a-jenkins-spring-clean) task to check for the latest stable version, and update if necessary. Major versions only come every ~3 months so it shouldn't be too onerous

Reply:
Good idea :+1: